haystack/pipelines/standard_pipelines.py

```diff
@@ -596,7 +596,10 @@ def __init__(
         if isinstance(pipeline, QuestionAnswerGenerationPipeline):
             setattr(output_translator, "run", output_translator.run_batch)
 
-        graph = pipeline.pipeline.graph
+        if hasattr(pipeline, "pipeline"):
+            graph = pipeline.pipeline.graph
+        else:
+            graph = pipeline.graph
```

Comment:
Hey @nickchomey ! I think @sjrl was suggesting this change:

```suggestion
            graph = pipeline.graph  # type: ignore
```

The PR is fine by the way! Thank you for debugging it :+1: